### ISSUE TITLE
Change unified check in design page wireframes to 320

### ISSUE THREAD

**Issue by loripusey** [department-of-veterans-affairs/va.gov-team#65448]
## Background
I spoke with Matt Dingee about mobile wireframe resolutions. After discussing it more, DSC's thinking evolved to...we should be testing to 400x zoom on desktop (for accessibility purposes) and that results in a 320px viewport width once you zoom in that far. So if your mobile design works at 320px, the desktop version will work responsively when zoomed in 400 times. DSC is recommending designing at 320 and not 375. Based upon this and the other points Ben mentioned, can we discuss changing your design debt ticket to making everything 320?

[Slack thread](https://dsva.slack.com/archives/C02GXKL8WM6/p1694546982585149)

Ben and Zach talked this through a bit more, here's what this ticket involves: 

## Tasks

- [x] In the unified check in sketch file, only the Design page (and Symbols page) and working within the Design page, update all artboards to 320, which includes the following:
- [x] Create dupe symbols and set the dupes to 320
- [x] Then, add those 320 components into the existing artboards (don't replace the existing artboards)
- [x] Fix spacing, etc., that will be needed based on these changes to each artboard
- [ ] Validate with ux team 
- [ ] Once above are complete, update all Design - Priority pages

## AC

- [ ] Check boards are accurate on Design page
- [ ] Check boards are accurate on Design - Priority pages
- [ ] Check if boards are still all accurate and don't need fixing on some of our feature pages, especially the Priority pages

**Comment by benbrasso-agile6** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Check-in-Experience-61fc23a2cb8a14001132e102/issues/department-of-veterans-affairs/va.gov-team/65448?planning-poker) with Zenhub @cgednalske @kellysmith1008 @ytsaoca @zach-park

**Comment by zach-park** [comment]
Added "update all Design - Priority pages" tot he AC. Started working on them.

**Comment by benbrasso-agile6** [comment]
@ytsaoca look through the commend thread on this ticket of Zach’s. Not sure if there’s anything you feel comfortable completing on it, but I’ll go ahead and add you as an assignee, just in case - https://app.zenhub.com/workspaces/check-in-experience-61fc23a2cb8a14001132e102/issues/gh/department-of-veterans-affairs/va.gov-team/65448

Also, tagging @cgednalske

**Comment by benbrasso-agile6** [comment]
We determined that we don't need to finish the rest of the 320 resizing for next sprint @loripusey 

Team - let's just work on validating the Design page (if that hasn't already been validated already)

**Comment by cgednalske** [comment]
I took a look at the design page yesterday and it looked good to me 👍 But wasn't sure if I was actually supposed to validate it yet 😄

**Comment by zach-park** [comment]
@benbrasso-agile6 @cgednalske @ytsaoca , 
Completed the Design page and the following Piority pages (Send out 45 min check-in text, #1, #2, #3 ) and the remaining Priority 4-9 pages are descoped.

**Comment by benbrasso-agile6** [comment]
K, let me do a double check to validate

**Comment by benbrasso-agile6** [comment]
@zach-park https://sketch.com/s/0e890de3-2530-4ee0-986e-cf0314334aec/a/bLgZZaV?annotation=4eb6c0db-3c33-46c8-9558-19d11545e2b6

**Comment by benbrasso-agile6** [comment]
LGTM. Reminder to the team, 320 moving forward even if it's adding to priorities 4-9.
